tools/ci_build/build.py

```diff
@@ -2010,8 +2010,12 @@ def main():
     if args.build:
         if args.build_wheel:
             nightly_build = bool(os.getenv('NIGHTLY_BUILD') == '1')
+            is_cpu_only = not(args.use_cuda or args.use_dnnl or args.use_tensorrt
+                              or args.use_openvino or args.use_nuphar or args.use_vitisai
+                              or args.use_acl or args.use_armnn or args.use_dml)
+            # TODO: Each of the EPs should have a unique name suffix
             wheel_name_suffix = args.wheel_name_suffix
-            if not args.use_openmp and wheel_name_suffix is None:
+            if not args.use_openmp and wheel_name_suffix is None and is_cpu_only:
```

Comment:
wouldn't it have been better and easier to do this in setup.py ? all the package naming logic is contained there.

Reply:
Good suggestion. I reverted this change and will handle it in a separated PR.